spec/yaml/swiftnav/sbp/file_io.yaml

```diff
@@ -35,8 +35,12 @@ definitions:
       MSG_FILEIO_READ_RESPONSE message where the message length field
       indicates how many bytes were succesfully read. If the message is
       invalid, a followup MSG_PRINT message will print "Invalid fileio
```

Comment:
Do you think we should remove the documentation around the `MSG_PRINT` response? Even if its true right now we don't want people to rely on it as part of the spec.